backend/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
`package.json` and `package-lock.json` should be located in the general directory.